pysb/bng.py

```diff
@@ -23,6 +23,13 @@
 except ImportError:
     pass
 
+# Python 2
```

Comment:
Would rather see a single line comment explaning the whole block... the 2 / 3 comments are not helpful for someone who doesn't know why this is being done. Something like, "Alias basestring under Python 3 for forwards compatibility".